aidial_adapter_bedrock/llm/converse/types.py

```diff
@@ -0,0 +1,123 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Literal, TypedDict, Union
+
+from aidial_adapter_bedrock.utils.json import remove_nones
+from aidial_adapter_bedrock.utils.list_projection import ListProjection
+
+
+class ConverseRole(str, Enum):
```

Comment:
It would have been great to use the types from `boto3-stubs`: https://youtype.github.io/boto3_stubs_docs/mypy_boto3_bedrock_runtime/

instead of defining our own. Please create an issue for this.